### DISCUSSION TITLE
x-mask:dynamic input takes focus whenever Livewire property updates

### DISCUSSION THREAD

**Discussion by BrianLangevin** [alpinejs/alpine#2890] [7. Bugs (archived)]
This issue is occurring on iOS Safari.

My Livewire component has multiple inputs bound by wire:model. One input has an x-mask:dynamic

Whenever one of the inputs sends a network request to update its Livewire component value, the x-mask input becomes focused.

**Comment by jottenlips** [comment]
I am experiencing this in our django app. On iOS the x-mask input will focus and wont unfocus

**Comment by mitchellmusarra** [comment]
@BrianLangevin were you able to find a solution or workaround to this problem?

**Comment by joaokamun** [comment]
This happens on my MacOS Safari as well.

MacOS: 13.1 
Safari: 16.2
